.circleci/config.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           POSTGRES_USER: test
           POSTGRES_DB: test
     environment:
+      POSTGRES_USER: test
+      POSTGRES_DB: test
```

Comment:
I'm kinda sad that these variables are not passed to the containers, but nothing we can about it.

Reply:
will be better if these variables used to configure tests - https://github.com/cossacklabs/acra/blob/master/.circleci/integration.sh#L7
I thought that tests rely on them, but not. To sync them we need pass same values to test

Reply:
> will be better if these variables used to configure tests

I guess we can't just take exported variables from script executed in one container and apply them to another container. Also, database containers start before our ones (be it `android-build` or `ci-py-go-themis`). So I guess we shall duplicate credentials for every job: one user+password+... in DB container env and one (with generic names such as `TEST_DB_USER`) in our container env. What do ya think?